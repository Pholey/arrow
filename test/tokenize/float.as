5.2310
7523.0
214e+10
325e1
5e-1243
325.235e1
