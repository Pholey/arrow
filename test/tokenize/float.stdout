test/tokenize/float.as:1:1-7: float: 5.231
test/tokenize/float.as:2:1-7: float: 7523
test/tokenize/float.as:3:1-8: float: 2.14e+12
test/tokenize/float.as:4:1-6: float: 3250
test/tokenize/float.as:5:1-8: float: 5e-1243
test/tokenize/float.as:6:1-10: float: 3252.35
test/tokenize/float.as:7:1: end
